native/start.sh

```diff
@@ -0,0 +1,2 @@
+/home/stephen/dev/Zilliqa/native/venv/bin/python /home/stephen/dev/Zilliqa/native/main.py -n 6 -d 5 -l 1 --port 30303 --websocket= --transaction-sender=0 --ds-guard=4 --shard-guard=0 --bucket=zilliqa-devnet --conf-dir=/home/stephen/dev/testnet/localdev --origin-server=/home/stephen/dev/Zilliqa/tmp --multiplier-fanout=1 --out-dir=/home/stephen/dev/Zilliqa/tmp --testnet=localdev --build-dir=/home/stephen/dev/Zilliqa/build --skip-non-guard-ds
```

Comment:
There are funnily enough quite a lot of stephens in the world, this will be removed to satisfy the non stephens and the files removed as they are not intended for this release.